package.json

```diff
@@ -2,9 +2,29 @@
   "name": "idb",
   "version": "6.1.5",
   "description": "A small wrapper that makes IndexedDB usable",
-  "main": "build/cjs/index.js",
-  "module": "build/esm/index.js",
-  "typings": "build/esm/index.d.ts",
+  "main": "build/index.cjs",
+  "module": "build/index.js",
+  "types": "build/index.d.ts",
+  "exports": {
+    ".": {
+      "module": "./build/index.js",
+      "import": "./build/index.js",
+      "default": "./build/index.cjs"
```

Comment:
can be simplified to:
```
      "require": "./build/index.cjs",
      "default": "./build/index.js"
```

Reply:
@developit I think you recommended using `"module"` fields in the past. Are they no longer needed?

Reply:
Using the module field works differently than require/default - in Webpack and Rollup, it will allow require() to resolve to the ESM bundle rather than forcing it to resolve to the CJS one. This avoids the module being bundled twice when mixed require + import is used.

Reply:
Cheers!